src/app/core/data/social-media-list.ts

```diff
@@ -1,28 +1,43 @@
+// (Specifically for Landing Page)
 export const socialMediaLists: socialLists[] = [
   {
-    name: 'Facebook',
+    name: 'facebook',
     path: 'https://www.facebook.com/pages/category/Photography-Videography/Wiins-109652504066491/',
-    src: '../../../../../assets/img/core/icon/Facebook.svg',
+    src: '../../../assets/img/core/icon/facebook.svg',
   },
   {
-    name: 'Instagram',
+    name: 'instagram',
     path: 'https://www.instagram.com/wiins___/',
-    src: '../../../../../assets/img/core/icon/Instagram.svg',
+    src: '../../../assets/img/core/icon/instagram.svg',
   },
   {
-    name: 'Linkedin',
+    name: 'linkedin',
     path: 'https://www.linkedin.com/company/espace-wiin-s/',
-    src: '../../../../../assets/img/core/icon/Linkedin.svg',
+    src: '../../../assets/img/core/icon/linkedin.svg',
   },
   {
-    name: 'Twitter',
+    name: 'twitter',
     path: 'https://twitter.com/espace_s',
-    src: '../../../../../assets/img/core/icon/Twitter.svg',
+    src: '../../../assets/img/core/icon/twitter.svg',
   },
   {
-    name: 'Github',
+    name: 'github',
     path: 'https://github.com/etsraphael/WiinsWebDapp',
-    src: '../../../../../assets/img/core/icon/Github.svg',
+    src: '../../../assets/img/core/icon/github.svg',
+  },
+];
+
+// (Specifically for Coming Soon)
+export const socialIconsLists: socialLists[] = [
+  {
+    name: 'discord',
+    path: 'https://discord.gg/Kpdh29U6',
+    src: '../../../assets/img/coming-soon/discord.svg',
+  },
+  {
+    name: 'github',
+    path: 'https://github.com/etsraphael/WiinsWebDapp',
+    src: '../../../assets/img/core/icon/github.svg',
```

Comment:
This link doesn't work due to the uppercase letter. 
All the files in ``src/assets/img/core/icon`` have the be update with a lowercase as a first letter

Reply:
file already in uppercase letter as we can see on this preview.
I'll try something about this bug